plugins/test/unit/plugins/db/test_models.py

```diff
@@ -1089,15 +1089,16 @@ def setUp(self):
         self.unit.repodata['other'] = '''
 <package arch="noarch" name="cat" pkgid="{{ pkgid }}">
     <version epoch="0" rel="1" ver="1.0" />
+    <changelog>Sometimes contains {{ template vars }} description</changelog>
 </package>'''
         self.unit.repodata['primary'] = '''
 <package type="rpm">
   <name>cat</name>
   <arch>noarch</arch>
   <version epoch="0" rel="1" ver="1.0" />
   <checksum pkgid="YES" type="{{ checksumtype }}">{{ checksum }}</checksum>
-  <summary>A dummy package of cat</summary>
-  <description>A dummy package of cat</description>
+  <summary>A dummy {{ var }} package of cat</summary>
+  <description>A dummy package of cat with some description of {{ template var}}</description>
```

Comment:
These get added to a test XML doc, but don't appear to get tested. Should they be?